packages/aws/src/aws.module.ts

```diff
@@ -35,6 +35,14 @@ export const AwsModule: ModuleInterface = {
                 new EnvironmentVariableResolver("AWS_REGION"),
             ]
         },
+        {
+            parameterName: AwsModuleKeyname + ".serviceTracing.isActive",
```

Comment:
I was debating using this new field or the `PRISTINE_AWS_XRAY_DEBUG` that we already have in the tracing and xray modules